check_required_packages.sh

```diff
@@ -17,15 +17,16 @@ fi
 
 function get_required_packages() {
     case "${platform,,}" in
-    "centos" | "redhat" | "rocky")
+    "centos" | "rhel" | "rocky")
         required_packages=('fontconfig' 'libX11' 'libxkbcommon-x11' 'xcb-util-renderutil' 'libglvnd-egl' 'libglvnd-opengl' 'libxkbcommon')
         ;;
     "ubuntu")
         required_packages=('fontconfig' 'libegl1' 'libopengl0' 'libxkbcommon0' 'libxcb-xinput0')
         ;;
     *)
-        echo "WARNING: List of dependencies for distribition \"$platform\" is unknown."
+        echo "WARNING: List of dependencies for distribution \"$platform\" is unknown."
         echo "Please install libX11 and/or libfontconfig packages."
+        exit
```

Comment:
I think exit is 0 by default; should this exit with a nonzero code?

Reply:
No, I want the script to not proceed to next steps if we don't have the packages specified for that platform. For e.x. we don't have required packages for `suse` yet but we [support](https://www.schrodinger.com/supportedplatforms) it.

Reply:
I meant - should this be `exit 1` instead?

Reply:
Oh I see.

May be it also makes sense to print out https://www.schrodinger.com/supportedplatforms.